flyteadmin/pkg/manager/impl/validation/named_entity_validator.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/flyteorg/flyte/flyteidl/gen/pb-go/flyteidl/core"
 )
 
-var archivableResourceTypes = sets.NewInt32(int32(core.ResourceType_WORKFLOW), int32(core.ResourceType_TASK))
+var archivableResourceTypes = sets.NewInt32(int32(core.ResourceType_WORKFLOW), int32(core.ResourceType_TASK), int32(core.ResourceType_LAUNCH_PLAN))
```

Comment:
can you verify with this change if this resolves https://linear.app/unionai/issue/COR-2094/support-archiving-launchplans?

we'll need to possibly make uctl/flytectl changes and definitely UI ones but it would be good to update the ticket with this context

Reply:
drop a comment there. Thank you!